assets/scss/custom/plugins/icons/_google-material.scss

```diff
@@ -0,0 +1,15 @@
+// Overwriting the theme stylesheet to fix font path
+
+// Google Material Icon Font via
+// https://github.com/marella/material-symbols/tree/main/material-symbols
+
+@font-face {
+  font-family: 'Material Symbols Outlined';
+  font-weight: 400 700;
+  font-display: block;
+  font-style: normal;
+  src: local('Material Icons'), local('MaterialIcons-Outlined'),
+    url('../docs/fonts/material-symbols-outlined.woff2') format('woff2'),
+    url('../../docs/fonts/material-symbols-outlined.woff2') format('woff2'),
+    url('../../../docs/fonts/material-symbols-outlined.woff2') format('woff2'),
```

Comment:
The preview URL shows 404 errors for the font in the network tab. This should not be done this way. font-face src is not intended as a trial and error method to lookup fonts in various paths.

![image](https://github.com/user-attachments/assets/ec52ca6d-8423-4372-ad03-095725ee5ec8)

Reply:
Don't know why this test version was shown here. I updated this commit shortly after creation. I triggered it again with a forced git push. That seemed to update it.